github/misc.go

```diff
@@ -107,19 +107,19 @@ func (c *Client) ListCodesOfConduct(ctx context.Context) ([]*Conduct, *Response,
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
 
-	var conducts []*Conduct
-	resp, err := c.Do(ctx, req, &conducts)
+	var codeOfConducts []*CodeOfConduct
+	resp, err := c.Do(ctx, req, &codeOfConducts)
 	if err != nil {
 		return nil, resp, err
 	}
 
-	return conducts, resp, nil
+	return codeOfConducts, resp, nil
 }
 
-// ListCodeOfConductByKey returns an individual code of conduct.
+// GetCodeOfConductByKey returns an individual code of conduct.
 //
 // https://developer.github.com/v3/codes_of_conduct/
-func (c *Client) ListCodeOfConductByKey(ctx context.Context, key string) (*Conduct, *Response, error) {
+func (c *Client) GetCodeOfConductByKey(ctx context.Context, key string) (*CodeOfConduct, *Response, error) {
```

Comment:
updated. also i am curious what would be the better naming convention if there are more than one way to get a code of conduct.

GetCodeOfConductByKey(..., key string) or GetCodeOfConductBy(..., key string) (since key is required input)